app/src/main/java/dev/itsvic/parceltracker/api/Core.kt

```diff
@@ -44,6 +44,7 @@ enum class Service {
     SAMEDAY_RO,
     UKRPOSHTA,
     POSTNORD,
+    HERMES,
```

Comment:
please put this in alphabetical order

Reply:
this is still the case as of 8e0b94e78ca28f68eff2c553d6c30eed769f6b2e, it's only been moved in the `deliveryService` map. please move it in the service enum too, as that's the list that's rendered in the UI

Reply:
yeah, makes sense